js/models/SMTPSettings.js

```diff
@@ -1,9 +1,50 @@
 import BaseModel from './BaseModel';
+import app from '../app';
+import is from 'is_js';
 
 export default class extends BaseModel {
   defaults() {
     return {
-      Notifications: true,
+      notifications: false,
+      serverAddress: '',
+      username: '',
+      password: '',
+      senderEmail: '',
+      recipientEmail: '',
     };
   }
+
+  validate(attrs) {
+    const errObj = {};
+    const addError = (fieldName, error) => {
+      errObj[fieldName] = errObj[fieldName] || [];
+      errObj[fieldName].push(error);
+    };
+
+    if (attrs.notifications) {
+      if (is.not.url(attrs.serverAddress.trim())) {
+        addError('serverAddress', app.polyglot.t('smtpModelErrors.serverAddress'));
+      }
+      if (is.empty(attrs.username.trim())) {
+        addError('username', app.polyglot.t('smtpModelErrors.username.caseEmpty'));
+      } else if (is.not.alphaNumeric(attrs.username.trim())) {
+        addError('username', app.polyglot.t('smtpModelErrors.username.caseInvalid'));
+      }
+      if (is.empty(attrs.password.trim())) {
+        addError('password', app.polyglot.t('smtpModelErrors.password'));
+      }
+      if (is.not.email(attrs.senderEmail.trim())) {
+        addError('senderEmail', app.polyglot.t('smtpModelErrors.senderEmail'));
+      }
+      if (is.not.email(attrs.recipientEmail.trim())) {
+        addError('recipientEmail', app.polyglot.t('smtpModelErrors.recipientEmail'));
+      }
+    }
```

Comment:
@jjeffryes This is not happening because of cross-tab errors. We specifically avoid this happening by having each tab work on a clone of the model, so they're not working on the same model and any errors on a form in one tab, do not interfere nor are included with the saving of another tab.

The reason you are getting the error is because you have invalid data saved on the server. This happened because a.) we didn't have the right client validations in place b.) the server is not validating the settings data.

Cris has already fixed A and for B, I've logged a [server bug](https://github.com/OpenBazaar/openbazaar-go/issues/252).

Anyhow, to get you're data corrected, just click Save from the Advanced Tab and then refresh the app.